components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -356,7 +356,8 @@ function matchFunctions.getExtraData(match)
 		lastgame = Variables.varDefault('last_game'),
 		mapveto = matchFunctions.getMapVeto(match),
 		mvp = matchFunctions.getMVP(match),
-		isconverted = 0
+		isconverted = 0,
```

Comment:
is this actually needed?

Reply:
I am unsure, It was included in the initial match2 write by @Rathoz but i have been unable to find where its instantiated

Reply:
It's not, it was ported as is from Rocket League in the Dark Ages